guidelines/style-guidelines.md

```diff
@@ -1,3 +1,5 @@
+The following are guidelines, not rules. Assess every situation case-by-case and use your judgement.
```

Comment:
Note: Go through and fix all the lists (full stop on the last point) when the other changes are confirmed.